services/brig/src/Brig/API/Client.hs

```diff
@@ -163,3 +169,10 @@ legalHoldClientRequested (LegalHoldClientRequest requester targetUser lastPrekey
     eventData = LegalHoldClientRequestedData requester targetUser lastPrekey' clientId
     lhClientEvent :: ClientEvent
     lhClientEvent = LegalHoldClientRequested eventData
+
+removeLegalHoldClient :: UserId -> AppIO ()
+removeLegalHoldClient uid = do
+    clients <- Data.lookupClients uid
+    -- Should only be one; but just in case we'll treat it as a list
+    let legalHoldClients = filter ((== LegalHoldClientType) . clientType) clients
+    forM_ legalHoldClients  (execDelete uid Nothing)
```

Comment:
I don't know if there should be a warning in the logs if there is ever more than one LH client?  Or even an error?

Reply:
We have tests which assert this; and in this case it doesn't actually cause any problems (and it's nicer to write), so is it cool if I just leave it?